server/src/routes/api/v1/reviewsRouter.js

```diff
@@ -0,0 +1,21 @@
+import express from "express";
+import { Review } from "../../../models/index.js";
+
+const reviewsRouter = new express.Router();
+
+reviewsRouter.delete("/:id", async (req, res) => {
+  const id = req.params.id;
+  const userId = req.user.id;
+  const reviewUserId = req.body.reviewUserId;
+
+  if (userId === reviewUserId) {
+    try {
+      await Review.query().deleteById(id);
+      res.status(200).json({ message: "Review successfully deleted" });
+    } catch (err) {
+      res.status(500).json({ error: err });
+    }
+  }
```

Comment:
Im being a stickler, but this is important. Currently, if the userId and reviewUserId do not match, the server wont respond, which will lead to a network timeout error 

You need to provide an else, that returns a status of 401 (unauthorized). Thats it!